azmenak/react-stripe-checkout
# add testing

test: add minimum viable testing framework 

- include mocha and enzyme for unit tests
- add nyc for code coverage
- add tests to ensure token and key are passed to stripe
